### PR TITLE
Mhp 2412 migrate mobile auth

### PR DESCRIPTION
**Pull request by jesusfreak3**

### CODE REVIEW

File: src/components/authentication/authLanding.component.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import template from './authLanding.html';
+
+angular.module('missionhubApp').component('authLanding', {
+    controller: authLandingController,
+    template: template,
+    bindings: {
+        accessToken: '<',
```
Comment (OzzieOrca) [inline src/components/authentication/authLanding.component.js]: What's this for?

File: src/components/authentication/authLanding.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<div class="m-auto tc" >
+    <img src='./HappyBlueLoop.gif'>
+    <p class='f3 pa3 bg-warning white w-50 dib' ng-if="$ctrl.inviteState">
```
Comment (OzzieOrca) [inline src/components/authentication/authLanding.html]: And the `ng-if="$ctrl.inviteState"`?

File: src/components/authentication/authLanding.component.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    },
+});
+
+function authLandingController() {}
```
Comment (OzzieOrca) [inline src/components/authentication/authLanding.component.js]: Does this need a controller?

**Comment by jesusfreak3** [comment]
@OzzieOrca dude man guy... how did i let all that slip through.....

**Pull request merged**